.github/workflows/opensearch-observability-test-and-build-workflow.yml

```diff
@@ -9,15 +9,19 @@ env:
 
 jobs:
   build:
+    strategy:
+      matrix:
+        java: [8, 11, 14, 17]
```

Comment:
remove matrix if you are not using it in this workflow?

Reply:
I'll need to make changes to correctly use the matrix versions but we need to have it as per the release campaign

Reply:
got it. forgot about that i thought it's only to change default version